tests/broker-test.py

```diff
@@ -71,7 +71,7 @@ def test_broker_username_password(scopes, expected_token_type):
     print("Testing broker username password flows by using accounts in local .env")
     username = os.getenv("BROKER_TEST_ACCOUNT") or input("Input test account for broker test: ")
     password = os.getenv("BROKER_TEST_ACCOUNT_PASSWORD") or getpass.getpass("Input test account's password: ")
-    assert(username and password, "You need to provide a test account and its password")
+    assert username and password, "You need to provide a test account and its password"
     result = pca.acquire_token_by_username_password(username, password, scopes)
     _assert(result, expected_token_type)
     assert(result.get("token_source") == "broker")
```

Comment:
Thanks for the catch, @emmanuel-ferdman !

While you are already at it, would you please also modify this similar line? (I tried to do that but then I do not seem to have permission to push to your branch.)

P.S.: CCing to @fengga for awareness of the right usage of Python's native `assert`.

```suggestion
    assert result.get("token_source") == "broker"
```

Reply:
@rayluo yes, thanks! added the suggestion 😄